src/App.js

```diff
@@ -1,25 +1,29 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+/* eslint-disable no-unused-vars */
+import React, { Component } from 'react';
+import {BrowserRouter, Route, Switch} from 'react-router-dom';
 
-const App = () => (
-  <div className="App">
-    <header className="App-header">
-      <img src={logo} className="App-logo" alt="logo" />
-      <p>
-        <code>Author Haven</code>
-      </p>
-      <a
-        className="App-link"
-        href="https://reactjs.org"
-        target="_blank"
-        rel="noopener noreferrer"
-      >
-        test
-      </a
-      >
-    </header>
-  </div>
-);
+import Home from './views/HomeView';
+import Login from './views/LoginView';
+import Error from './views/ErrorView';
+import Navigation from './views/NavigationView';
+
+
+class App extends Component {
+	render() {
+		return (
```

Comment:
Why is the indentation like this?

Reply:
it is set by eslint, the indentation is set to 1 tabspace. But I have to reformat it to 2 white spaces.